tdrs-backend/tdpservice/parsers/parse.py

```diff
@@ -119,7 +117,7 @@ def bulk_create_records(unsaved_records, line_number, header_count, datafile, df
 
                 try:
```

Comment:
nit: we should try merging this try with the upper try and catch specific exception. No need to have two tries.